relm-core/examples/main.rs

```diff
@@ -145,22 +177,39 @@ fn main() {
         }
     }
 
+    // Now we're going to have some fun with futures and the `futures_glib` crate.
+    // First we need some boilerplate code to set up the event loop.
```

Comment:
The boilerplate code is actually to create an `Executor` that allows to execute (`spawn`) futures on the glib event loop.